### PR TITLE
fixes various flaky tests

### PR DESCRIPTION
**Pull request by nionis**
Fixes various flaky tests which randomly break

### CODE REVIEW

File: apps/discovery-platform/src/registered-node/index.spec.ts [hunk 1]
```diff
@@ -169,7 +169,7 @@ describe("test registered node functions", function () {
   it("should keep updated_at updated", async function () {
     await createRegisteredNode(dbInstance, mockNode("1"));
     const node = await getRegisteredNode(dbInstance, "1");
-    await wait(1);
+    await wait(100);
```
Comment (esterlus) [inline apps/discovery-platform/src/registered-node/index.spec.ts:172]: So what's the actual problem here?

Does wait not work correctly? Or why are the timestamps not different with 1ms difference?
Whats the granularity of the db timestamps?

**Review by esterlus — COMMENTED** [review]
no changeset needed

File: apps/discovery-platform/src/registered-node/index.spec.ts (hunk 1 above)
Comment (nionis) [reply apps/discovery-platform/src/registered-node/index.spec.ts:172]: `wait` uses `setTimeout` which is not entirely accurate [mentioned in this section](https://nodejs.org/dist/latest-v17.x/docs/api/timers.html#settimeoutcallback-delay-args) - so relying on just 1ms wait could lead to this problem
DB granularity is to the millisecond

Comment (esterlus) [reply apps/discovery-platform/src/registered-node/index.spec.ts:172]: Ok can we do with `wait(2)` then?

Comment (nionis) [reply apps/discovery-platform/src/registered-node/index.spec.ts]: makes sense, should not be a problem

**Review by esterlus — APPROVED** [review]
Please add a comment explaining that `wait(1)` might be less than 1 because of timer inaccuracies before merging.

**Pull request merged**